Emby.Server.Implementations/ScheduledTasks/Tasks/RefreshMediaLibraryTask.cs

```diff
@@ -58,6 +71,8 @@ public Task Execute(CancellationToken cancellationToken, IProgress<double> progr
 
             progress.Report(0);
 
+            _imageGenerator.GenerateSplashscreen(Path.Combine(_applicationPaths.DataPath, "splashscreen.webp"));
```

Comment:
Will this always serve webp regardless of browser support?

Reply:
No, this goes through the normal image return process. I just tested it and got image/png